assets/chef-recipes/cookbooks/mariadb-maxscale/recipes/install_maxscale.rb

```diff
@@ -129,9 +129,13 @@
   mode "640"
 end
 
-check_version 'Check the installed version of the MaxScale server' do
-  version node['maxscale']['version']
-  deb_package_name 'maxscale'
-  rhel_package_name 'maxscale'
-  suse_package_name 'maxscale'
+if node['maxscale']['ci_product']
```

Comment:
Use `not_if` or `only_if` property of the resource to initiate it, do not wrap it into the `if` clause.